### ISSUE TITLE
traefik reverse proxy does no longer start

### ISSUE THREAD

**Issue by markusschuh** [devonfw/my-thai-star#280]
Category: Bug

Severity: Low

Expected behaviour:
`docker-compose -f docker-compose.traefik.yml up -d` from git checkout should start a docker container "reverse-proxy" that starts a running traefik.

Actual/current behaviour:
Logs of "my-thai-star_reverse-proxy_1" container contain error message "command traefik error: flag needs an argument: -docker"

Steps to reproduce:
Checkout git-repository, ensure access to a running docker environment, run `docker-compose -f docker-compose.traefik.yml up -d`

Description of issue:
The `docker-compose.traefik.yml` does not specify a tag for traefik image. It was written, when v1.7 was the latest traefik version. Now this has changed to v2.0.
Proposed solution:  Use  tag "v1.7" or "v1.7-alpine" instead of  no tag at all inside `docker-compose.yml`  Do not use v1.7.16 or alike.
This is a common practice:  Don't refer to any latest, but refer to a "latest" patch version (contains fixes) for a specific major version. This avoids issues in command line syntax / interface / ... changes during changes from one major version to another.

**Comment by dario-rodriguez** [comment]
I think that this isssue is solved with the PR #280.

@markusschuh Can I close this issue or is there anything peding?

**Comment by markusschuh** [comment]
This was addressed by PR #288. I missed to link from there to here.

There is just one minor topic open:  When I've created the PR, traefik team had not yet published a "2.0" tagged image on docker hub. This was now added. So there is one last step useful - but this needs another PR, doesn't it?

WAS:     image: traefik
AS-IS:    image: traefik:2.0.0
SHOULD-BE:  image:  traefik:2.0

A tag like "2.0" - which refers to the latest patched published 2.0.x - ensures security fixes are included, if a fresh container is loaded from github - but features do not break.
